qsirecon/tests/test_interfaces.py

```diff
@@ -0,0 +1,47 @@
+"""Command-line interface tests."""
+
+from pathlib import Path
+
+import nibabel as nb
+import numpy as np
+import pytest
+
+from qsirecon.interfaces.gradients import GradientSelect
+from qsirecon.tests.utils import download_test_data
+
+
+@pytest.mark.integration
```

Comment:
I wouldn't call this an integration test. I normally reserve that for the tip-to-tail workflow tests.